doc/documentation.rst

```diff
@@ -79,4 +79,21 @@ directory to the inferred package:
    (documentation
     (mld_files :standard \ wip))
 
+.. _odoc-options:
+
+Passing options to Odoc
+=======================
+
+.. code-block:: lisp
+
+    (env
+     (<profile>
+      (odoc <optional-fields>)))
+
+See :ref:`dune-env` for more details on the ``(env ...)`` stanza.
+``<optional-fields>`` are:
+
+- ``(warn-error <bool>)`` turns warnings into errors, if set to ``true``.
+  This field is available since Dune 2.2 and requires Odoc 1.4.3.
```

Comment:
Also, I'm not too sure about the `warn-error` key. It mimics the configuration of the ocaml compilers, but that part is confusing.

What do you think about `(warnings fatal)` and `(warnings nonfatal)` (or maybe `(warnings display)`), which can be extended more easily to things like `(warnings silenced)` if we decide to extend the feature later?

Reply:
Just did that. I agree it looks better.